### PR TITLE
Numerous CLI and reporting improvements:

### LINKED ISSUE #128: We should disable coloring in CLI if stdout is not a TTY

### LINKED ISSUE #168: Highlight browser list in reporters

### PR DESCRIPTION
**Pull request by inikulin**
- Higlight browser list in some reporters (closes #168)
- Always force colors for CLI and disable them for any report stream
  except stdout (closes #128)
- Introduce spinner for bootstrapping and test run (in case then report is
  redirected to file)

\cc @AlexanderMoskovkin @helen-dikareva 
@VasilyStrelyaev pls, check comments and strings

### CODE REVIEW

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by helen-dikareva** [comment]
lgtm

**Comment by inikulin** [comment]
Don't merge this, please. I have some additions.

**Comment by inikulin** [comment]
Added support for `--no-color` option

File: src/cli/remotes-wizard.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+export default async function (testCafe, remoteCount, showQRCode) {
+    var connections = [];
+
+    if (remoteCount) {
+        spinner.hide();
+
+        for (var i = 0; i < remoteCount; i++) {
+            var browserConnection = testCafe.createBrowserConnection();
+
+            console.log(`To connect remote browser #${i + 1} open ${chalk.underline.blue(browserConnection.url)}`);
```
Comment (VasilyStrelyaev) [inline src/cli/remotes-wizard.js]: To connect a remote browser \#${i + 1}, use it to open

File: src/cli/remotes-wizard.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+            if (showQRCode) {
+                console.log('or use this QR-code:\n');
```
Comment (VasilyStrelyaev) [inline src/cli/remotes-wizard.js]: or scan

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by inikulin** [comment]
FPR

**Comment by helen-dikareva** [comment]
lgtm

File: src/cli/remotes-wizard.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    if (remoteCount) {
+        spinner.hide();
+
+        for (var i = 0; i < remoteCount; i++) {
+            var browserConnection = testCafe.createBrowserConnection();
+
+            console.log(`To connect a remote browser #${i + 1}, use it to open ${chalk.underline.blue(browserConnection.url)}`);
+
+            if (showQRCode) {
+                console.log('or use scan this QR-code:\n');
```
Comment (VasilyStrelyaev) [inline src/cli/remotes-wizard.js]: oops

Comment (VasilyStrelyaev) [inline src/cli/remotes-wizard.js]:
```
                console.log('or scan this QR-code:\n');
```

Comment (inikulin) [inline src/cli/remotes-wizard.js]: yep

**Comment by inikulin** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**